frontend/src/lib/components/graph/util.ts

```diff
@@ -0,0 +1,15 @@
+export const NODE = {
+	width: 200,
+	height: 45,
+	gap: {
+		horizontal: 20,
+		vertical: 50
+	}
+}
+
+export function* createIdGenerator(): Generator<number, number, unknown> {
+	let id = 0
+	while(true) {
+		yield id++
```

Comment:
We have a logic in the flow to generate increasing ids with letters: 'a', b', ..., 'z', 'aa', 'ab
Can we use this to avoid casting ids ?